Plugin/CppApi/include/ToolManager.h

```diff
@@ -40,6 +40,10 @@ class TOOLKIT_EXPORT ToolManager {
 
   void addTool(AbstractTool* tool);
 
+  void removeTool(const QString& toolName);
```

Comment:
Do we want to have a version of this function that takes in an AbstractTool instance? I wonder if you'd want to ensure you are removing a specific instance of a tool and not just some instance of a tool that is in the tool manager (in the case that you have multiple AddLocalDataToolControllers, for example)

Reply:
I can certainly add that. At the moment the map only allows 1 tool per name so you can't have multiple AddLocalDataToolControllers...

Reply:
probably not necessary if that is how the map behaves. Is there a use case for having multiple instances of a tool controller? I was wondering this a while back. Maybe I want the Coordinate Conversion Tool (with UI) in my app, but then I want to also be able to directly use the Coordinate Conversion controller from the C++ side in a different tool?

Reply:
I'll add it anyway since it's essentially the lambda we run when a tool gets destroyed. I don't know about multiple tools. Tricky to see how that would work. I guess each could have a unique name/ID?